services/gateway/shared/utils.js

```diff
@@ -23,6 +23,14 @@ const transformParams = (type, params) => {
 	// TODO: Integrate this logic within the method defs instead
 	paramsKeys.forEach((paramKey) => {
 		let value = {};
+
+		if (params[paramKey].altSwaggerKey) {
+			data.push({
+				$ref: `#/parameters/${params[paramKey].altSwaggerKey}`,
+			});
+			return;
+		}
+
 		if (type === 'blocks' && paramKey === 'id') {
 			value = { $ref: '#/parameters/block' };
 		} else if (type === 'PoS' && paramKey === 'status') {
```

Comment:
Could you please set the `altSwaggerKey` for these special cases and entirely remove the `if...else` ladder here?

Reply:
Applied.

Reply:
Applied.